src/components/DivElement.js

```diff
@@ -1,6 +1,15 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
-
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super(content);
+    this.content = content;
+    this.tag = 'div';
```

Comment:
Another way you can do this is to call `super()` and pass both the content and tag. Think of calling `super()` as calling the parent class's constructor. It's like if I called `HTMLElement`'s constructor.

So here, you're passing `content` to the `HTMLElement`'s constructor as `tag`. It ends up working because you reassign content and tag, so keep this in mind for next time.